### PR TITLE
chore: update Jupyter image to use miniforge

### PR DESCRIPTION
**Pull request by o2a**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -35,8 +35,8 @@ RUN pip install --no-cache-dir ipywidgets \
     jupyter labextension install jupyter-leaflet
 
 # Add Dask Labextension
-RUN pip install --no-cache-dir dask_labextension && \
-    jupyter labextension install dask-labextension@3.0.0
+# JupyterLab 3.0 or greater - https://pypi.org/project/dask-labextension/
+RUN pip install --no-cache-dir dask-labextension
```
Comment (andrew-longmore-tessella) [inline Dockerfile:39]: Shouldn't we be specifying the version of the dask-labextension?

Comment (o2a) [reply Dockerfile:39]: Yes, we can pin the version. The "pip install" in the current image have not been pinned. It's the client side extension that was pinned at 3.0.0. JupyterLab 3.0.5 has been installed into the current image and the client side extension is not required with >JupyterLab 3.0. Looking at the Dockerfile further, all the other "pip installs" have not been pinned. Wondering if there is a reason for that....

Comment (o2a) [reply Dockerfile]: Updated the PR to pin the pip package versions

**Review by andrew-longmore-tessella — APPROVED** [review]

**Comment by iwalmsley** [comment]
Looks good, but worth checking if it's possible to create a Dask cluster and load the graphs in the way we've had issues with in the past?

**Comment by iwalmsley** [comment]
@o2a I think the above concern is handled by Andys testing which worked fine assuming the merge conflicts are resolved I think this can be merged.

**Pull request merged**